pm_uep_opt_write/pm_uep_opt_write.xml

```diff
@@ -0,0 +1,58 @@
+<tool id="pm_uep_opt_write" name="Generate uep outputs (pm-uep-opt)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package">pymuon-suite</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
+        unzip $folder &&
+        if test -f "params.yaml"; then echo "params.yaml present"; else echo "params.yaml missing" && exit 64; fi 
+        && if test -f "input.cell"; then echo "input.cell present"; else echo "input.cell missing" && exit 64; fi 
+        && cp $charge_density $charge_density.name &&
+        cp $castep_log $castep_log.name &&
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        find \$out_folder -type f -name "*.yaml" | xargs sed -i "s#^chden_path: .*#chden_path: .\/#g" &&
+        bash ${__tool_directory__}/run.sh \$out_folder &&
```

Comment:
pm-muairss writes both `chden_path` and `chden_seed` into the output files.

When you do the muonation with a YAML file with no `uep_chden` set, pm-muairss defaults `chden_path` to the working directory and `chden_seed` to the _name_ of the working directory. This can mismatch with whatever `.den_fmt` and `.castep` files you select in Galaxy, resulting in pm-uep-opt raising errors like:
```
no such file or directory: ./working.den_fmt
```

We should make sure the `chden_seed` matches the input files, e.g. `si` for `si.den_fmt`/`si.castep`.

Reply:
additional bug from this - run fails if den_fmt and castep have different names, e.g si.den_fmt and si-out.castep. changed castep file renaming to <den_filename>.castep, e.g. si.castep in the given example